sql/promscale--0.0.0.sql

```diff
@@ -0,0 +1,460 @@
+        , ('OPERATOR', '_prom_catalog.?(prom_api.label_array, ps_tag.tag_op_equals)')
+        , ('OPERATOR', '_prom_catalog.?(prom_api.label_array, ps_tag.tag_op_regexp_not_matches)')
+        , ('OPERATOR', 'prom_api.?(prom_api.label_array, prom_api.matcher_positive)')
+        , ('OPERATOR', 'prom_api.?(prom_api.label_array, prom_api.matcher_negative)')
+        , ('OPERATOR', 'prom_api.@>(prom_api.label_value_array, pg_catalog.text)')
+        , ('OPERATOR', 'prom_api.@>(prom_api.label_array, pg_catalog.jsonb)')
+        , ('OPERATOR', 'ps_tag.!==(pg_catalog.text, pg_catalog.anyelement)')
+        , ('OPERATOR', 'ps_tag.!==(pg_catalog.text, pg_catalog.text)')
+        , ('OPERATOR', 'ps_tag.!=~(pg_catalog.text, pg_catalog.text)')
+        , ('OPERATOR', 'ps_tag.#<(pg_catalog.text, pg_catalog.text)')
+        , ('OPERATOR', 'ps_tag.#<(pg_catalog.text, pg_catalog.anyelement)')
+        , ('OPERATOR', 'ps_tag.#<=(pg_catalog.text, pg_catalog.anyelement)')
+        , ('OPERATOR', 'ps_tag.#<=(pg_catalog.text, pg_catalog.text)')
+        , ('OPERATOR', 'ps_tag.#>(pg_catalog.text, pg_catalog.anyelement)')
+        , ('OPERATOR', 'ps_tag.#>(pg_catalog.text, pg_catalog.text)')
+        , ('OPERATOR', 'ps_tag.#>=(pg_catalog.text, pg_catalog.anyelement)')
+        , ('OPERATOR', 'ps_tag.#>=(pg_catalog.text, pg_catalog.text)')
+        , ('OPERATOR', 'ps_tag.==(pg_catalog.text, pg_catalog.anyelement)')
+        , ('OPERATOR', 'ps_tag.==(pg_catalog.text, pg_catalog.text)')
+        , ('OPERATOR', 'ps_tag.==~(pg_catalog.text, pg_catalog.text)')
+        , ('OPERATOR', 'ps_tag.@?(pg_catalog.text, pg_catalog.jsonpath)')
+        , ('OPERATOR', 'ps_trace.#(ps_trace.tag_map, ps_trace.tag_k)')
+        , ('OPERATOR', 'ps_trace.#?(ps_trace.tag_map, ps_trace.tag_k)')
+        , ('OPERATOR', 'ps_trace.?(ps_trace.tag_map, ps_tag.tag_op_not_equals)')
+        , ('OPERATOR', 'ps_trace.?(ps_trace.tag_map, ps_tag.tag_op_less_than)')
+        , ('OPERATOR', 'ps_trace.?(ps_trace.tag_map, ps_tag.tag_op_less_than_or_equal)')
+        , ('OPERATOR', 'ps_trace.?(ps_trace.tag_map, ps_tag.tag_op_greater_than)')
+        , ('OPERATOR', 'ps_trace.?(ps_trace.tag_map, ps_tag.tag_op_greater_than_or_equal)')
+        , ('OPERATOR', 'ps_trace.?(ps_trace.tag_map, ps_tag.tag_op_equals)')
+        , ('OPERATOR', 'ps_trace.?(ps_trace.tag_map, ps_tag.tag_op_regexp_not_matches)')
+        , ('OPERATOR', 'ps_trace.?(ps_trace.tag_map, ps_tag.tag_op_regexp_matches)')
+        , ('OPERATOR', 'ps_trace.?(ps_trace.tag_map, ps_tag.tag_op_jsonb_path_exists)')
+        ) x(objtype, objname)
+    )
+    LOOP
+        -- extension is installed into _prom_ext schema. thus cannot add it to the extension
+        IF NOT (_rec.objtype = 'SCHEMA' AND _rec.objname = '_prom_ext') THEN
+            EXECUTE format('ALTER EXTENSION promscale ADD %s %s', _rec.objtype, _rec.objname);
+        END IF;
+        EXECUTE format('ALTER %s %s OWNER TO %I', _rec.objtype, _rec.objname, '@extowner@');
```

Comment:
shouldn't this be to superuser not extowner? I also wonder why this wasn't caught by the tests.